packages/blade/src/components/Box/Box.tsx

```diff
@@ -12,7 +12,8 @@ const validateBackgroundString = (stringBackgroundColorValue: string): void => {
   if (__DEV__) {
     if (
       !stringBackgroundColorValue.startsWith('surface.background') &&
-      !stringBackgroundColorValue.startsWith('brand.')
+      !stringBackgroundColorValue.startsWith('brand.') &&
+      stringBackgroundColorValue !== 'transparent'
     ) {
       throwBladeError({
```

Comment:
add transparent here as well in the message??

Reply:
Done.